pages/common/sdcv.md

```diff
@@ -0,0 +1,37 @@
+# sdcv
+
+> StarDict console version. Command line dictionary client.
+> Dictionaries are provided separately.
+> More information: <https://manned.org/sdcv>.
+
+- Start sdcv interactively:
+
+`sdcv`
+
+- List installed dictionaries:
+
+`sdcv --list-dicts`
+
+- Display a definition from a specific dictionary:
+
+`sdcv --use-dict {{dictionary_name}} {{search_term}}`
+
+- Lookup definition with fuzzy search:
+
+`sdcv {{search_term}}`
+
+- Lookup definition with exact search:
+
+`sdcv --exact-search {{search_term}}`
+
+- Lookup definition and get json output:
+
+`sdcv -j {{search_term}}`
+
+- Look in specific directory for dictionaries:
+
+`sdvc --data-dir {{path/to/directory}} {{search_term}}`
+
+- Display help:
+
+`sdcv --help`
```

Comment:
I'm unsure of the usefulness of this example, since `--help` is pretty much a standard convention.

Reply:
most commands have it, its fine by me

Reply:
Agreed — if it's not good here it's probably not elsewhere and we should remove it across all the pages if we're going to here.

Reply:
I think most of the command line utilities accept the switch `-h` or `--help`. In my opinion we should only keep those that have anything apart from this (I do not have an example handy)

Reply:
Let's move this discussion to a new issue, how about.  It needs more attention than it will get in a random thread of a random PR, because it's quite common across the pages.

Reply:
Good idea @bl-ue, though I suspect it's really going to be a case-by-case basis (e.g. for `nmcli` it does make sense even though the example description isn't helpful since it also displays a list of subcommands).